ext/json/json.c

```diff
@@ -141,13 +141,14 @@ static PHP_MINFO_FUNCTION(json)
 }
 /* }}} */
 
-PHP_JSON_API int php_json_encode_ex(smart_str *buf, zval *val, int options, zend_long depth) /* {{{ */
+PHP_JSON_API int php_json_encode_ex(smart_str *buf, zval *val, int options, zend_long depth, zend_string *indent_str) /* {{{ */
```

Comment:
Would probably be a good idea to fix the return type from `int` to `zend_result` and propagate the change down the function defs as they only return SUCCESS or FAILURE.
But that could wait